src/components/SearchSelectMenu.tsx

```diff
@@ -59,7 +59,10 @@ export const SearchSelectMenu = ({
   ) : (
     <$MenuTrigger>
       {label ? <$WithLabel label={label}>{children}</$WithLabel> : children}
+      {/* <div style={{ display: 'flex' }}>
+        <div style={{ color: 'var(--color-red)' }}>Highest fees</div> */}
       <DropdownIcon isOpen={open} />
+      {/* </div> */}
```

Comment:
Remove commented code?

Reply:
man, my bad - i was working on some visual mock ups and i added everything on accident 😢